### ISSUE TITLE
setting break_reads_after_seconds parameter

### ISSUE THREAD

**Issue by JensUweUlrich** [nanoporetech/minknow_api#18]
Hi,

I sucessfully could set the break_reads_after_seconds parameter via Analysis_Configuration. After starting a playback run the behaviour of the sequencing suggests that the parameter is overruled by the value set in the sequencing TOML file. Is that correct? And if so, would it be possible to not overrule?

Thanks
Jens

**Comment by jorj1988** [comment]
If you want to tune the value of `break_reads_after_seconds`, you would have to modify the toml file itself - currently the protocol overwrites with its own configuration.

**Comment by JensUweUlrich** [comment]
Yes, that's the way I do it at the moment. But it would be nice if I could set the parameter via the minknow_api. This would avoid the user to set and reset the value of the parameter manually when using ReadUntil.
